SkillzNews/Message Cell/SkillzNewsMessageCell.js

```diff
@@ -20,7 +23,22 @@ export default class SkillzNewsMessageCell extends React.PureComponent {
     newsArticle: PropTypes.object.isRequired,
   }
 
+  constructor(props: any) {
```

Comment:
This should obviously be a stateless component, do you have any real need for state here, why are you using it?